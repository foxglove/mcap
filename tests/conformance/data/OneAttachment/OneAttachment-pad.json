{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {
      "type": "Attachment",
      "fields": [
        ["content_type", "application/octet-stream"],
        ["create_time", "1"],
        ["data", ["1", "2", "3"]],
        ["log_time", "2"],
        ["name", "myFile"]
      ]
    },
    {"type": "DataEnd", "fields": [["data_section_crc", "1702036232"]]},
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "1875167664"],
        ["summary_offset_start", "0"],
        ["summary_start", "0"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["pad"]}}
}
