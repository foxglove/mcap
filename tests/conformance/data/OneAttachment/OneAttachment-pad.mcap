version https://git-lfs.github.com/spec/v1
oid sha256:193f9ca28f574708c37c0466bd7044438c24ca9339e5e202d1dda5d6f8545f16
size 162
