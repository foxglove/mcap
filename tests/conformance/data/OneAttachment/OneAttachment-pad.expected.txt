Header library="" profile=""
Attachment content_type=application/octet-stream created_at=1 data=<010203> log_time=2 name=myFile
AttachmentIndex content_type=application/octet-stream data_size=3 length=81 log_time=2 name=myFile offset=28
Footer summary_crc=0 summary_offset_start=0 summary_start=125
